sbi/neural_nets/density_estimators/mixed_density_estimator.py

```diff
@@ -175,7 +175,7 @@ def log_prob_iid(self, x: Tensor, context: Tensor) -> Tensor:
         net_device = next(self.discrete_net.parameters()).device
         assert (
             net_device == x.device and x.device == context.device
-        ), f"device mismatch: net, x, context: {net_device}, {x.device}, {context.device}."
+        ), f"device mismatch: net, x, cont: {net_device}, {x.device}, {context.device}."
```

Comment:
i do not like this change, `cont` is really unintuitive imo (e.g. we also use it as `continuous` in mnle...)

Aside, all changes in this file will lead to merge conflicts for me and I am revising many things here, so I would suggest to just remove all your edits from this file.

Reply:
OK, I will remove these changes. But this will result in `ruff and hooks` failing on `main`. That's fine for now, but please fix line lengths in the PRs to come then.

Reply:
done.